scripts/local_testnet/setup_time.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+set -Eeuo pipefail
+
+source ./vars.env
+
+genesis_file=${@:$OPTIND+0:1}
```

Comment:
`OPTIND` doesn't do anything here as we're not using the `getopts` command to parse command line arguments, so this can be simplified to:

```suggestion
genesis_file=$1
```

Reply:
thank you!